R/zchunk_batch_modeltime.xml.R

```diff
@@ -0,0 +1,36 @@
+#' module_modeltime_batch_modeltime.xml
```

Comment:
It would be a source of confusion and doesn't really add value.  It is also simple to fix so I'll just switch `.xml.R` to '_xml.R` by both renaming this model time example and also so the chuck generator will generate them as such.